api/users/views.py

```diff
@@ -531,13 +532,15 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, NodesFilterMix
 
     ordering = ('-modified',)
 
+    @newrelic.agent.function_trace()
```

Comment:
From the new relic docs:

> function_trace does not give you a full profile of all the functions that happen in a decorated function.... If you need more detail about what is going on in that function, you would need to also apply the function trace to the function's child functions.

Is this why you've decorated both `get_default_queryset` and `get_queryset`?